articles/contributing/api-design-principles.md

```diff
@@ -0,0 +1,426 @@
+---
+title: Q# API Design Principles
+description: Q# API Design Principles
+author: cgranade
+ms.author: chgranad
+ms.date: 10/12/2018
```

Comment:
2 years ago? :)

Reply:
That's what I get for copying boilerplate headers. Thanks for catching that!